src/main.py

```diff
@@ -175,12 +187,32 @@ def cli():
 @cli.command()
 @option_disable_ssl_verification
 @click.argument("config", type=click.File("r"))
-def sync(config, disable_ssl_verification):
+@click.option("--project-id", type=int, help="The ID of your dbt Cloud project.")
+@click.option("--environment-id", type=int, help="The ID of your dbt Cloud environment.")
```

Comment:
Would it make sense to make those options `multiple=True` (like `--job-id`) to allow people to just select one project/env if they want but also select multiple ones if they want as well.

Also, we could add `[Optional]` in the help code to follow what we used in `--job-id`

Reply:
My thought process for the overall design decision is to identify the main entity at which the user would think of Jobs. We have dbt Account -> multiple projects -> and each project will have multiple environments. 
Customers always think of Jobs at a project level. Allowing the users to operate within a project scope gives better control, and the job Admins are sure that they are working only on that specific project and NOT updating jobs in a different project by mistake.

For Enterprise customers, if job admins have access to ~10-15 different projects, we should direct them to manage jobs only within a single project at a given point in time. I feel that having one project ID is a better design, keeping it simple and giving the user more control over the operational scope.
Also, I feel we should make project ID a mandatory CLI argument for the commands. Environment ID can be optional.

Moving on to environment ID parameter, We need to think about the question if customers would like to manage jobs across environments in a single command invocation OR would they want to do it per environment to have more control while invoking jobs-as-code? This would provide us the answer if we want to make environment ID multiple OR a single value.